glean_parser/templates/go_server.jinja2

```diff
@@ -157,7 +157,7 @@ func (g GleanEventsLogger) record(
 	if envelopeErr != nil {
 		panic("Unable to marshal log envelope to json")
 	}
-	fmt.Println(string(envelopeJson))
+	slog.Debug(string(envelopeJson))
```

Comment:
Why Debug? I'd rather leave this at info so it works out of the box for new services. If you prefer to have this logged at debug level we can add a template parameter to change the level case by case.

Reply:
Reverted to `fmt.Println`